tests/integration/helpers.py

```diff
@@ -14,8 +14,43 @@
 from utilities import retry
 
 
-async def remove_runner_bin(unit: Unit):
-    """Remove runner binary."""
+async def get_repo_policy_compliance_pip_info(unit: Unit) -> None | str:
+    """Get pip info for repo-policy-compliance.
+
+    Args:
+        source: The git source to install the package. If none the package is removed.
+    """
+    action = await unit.run("pip show repo-policy-compliance")
```

Comment:
Similar to above comments, GH action invokes `python3 -m pip`, the charm invokes `pip` with a full path, here it relies on pip being in `$PATH` - perhaps there can be one way of invocation?

Reply:
Switched to `python3 -m pip`